v2x/vlog_to_pbtype.py

```diff
@@ -629,12 +629,25 @@ def make_container_pb(
     for (driver_cell, driver_pin), sinks in interconn.items():
         if driver_cell in routing_cells:
             continue
-        for (sink_cell, sink_pin), path_attr in sinks:
-            if sink_cell in routing_cells:
-                continue
+
+        non_routing_sinks = [(sink, path_attr) for sink, path_attr in sinks
+                             if sink[0] not in routing_cells]
+
+        is_forking = len(non_routing_sinks) > 1
+
+        for (sink_cell, sink_pin), path_attr in non_routing_sinks:
+            attrs = dict(**path_attr)
+
+            # If a net forks remove pack pattern annotations from the branch
+            # that goes to an output port of the pb_type
+            if is_forking and sink_cell is None:
+                for a in ["pack", "PACK"]:
```

Comment:
We shoulds normalize the `pack` case somewhere else in the code so only one version needs to be checked here.

Reply:
Fixed